core-rust/state-manager/src/jni/mempool.rs

```diff
@@ -202,27 +202,27 @@ fn do_get_transactions_to_relay(
 // DTO Models + Mapping
 //
 
-/// Corresponds to the payload_hash
+/// Corresponds to HashCode from Java
 #[derive(Debug, PartialEq, Eq, Categorize, Encode, Decode)]
-pub struct JavaPayloadHash(Vec<u8>);
+pub struct JavaHashCode(pub Vec<u8>);
```

Comment:
Is this fine? In Java it's represented as `HashCode`, it feels better to have the same naming.